pages/common/elm.md

```diff
@@ -0,0 +1,15 @@
+# elm
+
+> Run and manage programs in Elm programming language.
+
+- Run interactive elm shell (repl):
+
+`elm --repl`
+
+- Initialize empty elm project:
+
+`elm --init`
+
+- Compile your project and serve it via local web server for easy development:
```

Comment:
Perhaps `Compile the current project and serve it via the local web server for easy development:` would read better?

Reply:
Thanks, going to fix it